docs/overview/getting-started/avalanche-consensus.md

```diff
@@ -2,21 +2,50 @@
 description: Consensus is the task of getting a group of computers to come to an agreement on a decision. Avalanche is a new consensus protocol that is scalable, robust, and decentralized.
 ---
 
-# Avalanche Blockchain Consensus
+# Consensus
 
-Consensus is the task of getting a group of computers to come to an agreement on a decision.
-Computers can reach a consensus by following a set of steps called a consensus protocol. Avalanche
+
+Consensus is the task of getting a group of nodes to come to an agreement on a decision.
+Nodes can reach a consensus by following a set of steps called a consensus protocol. Avalanche
 is a new consensus protocol that is scalable, robust, and decentralized. It has low latency and high
 throughput. It is energy efficient and does not require special computer hardware. It performs well
 in adversarial conditions and is resilient to "51% attacks." This document explains the Avalanche
-consensus protocol. The white-paper is [here.](https://www.avalabs.org/white-papers)
+consensus protocol. The white-paper is [here](https://www.avalabs.org/white-papers).
+
+![Consensus Comparison](/img/Consensus-protocol-comparison.png)
+
+Protocols in the Avalanche family operate through repeated sub-sampled voting. When a
+[validator](http://support.avalabs.org/en/articles/4064704-what-is-a-blockchain-validator) is
+determining whether a
+[transaction](http://support.avalabs.org/en/articles/4587384-what-is-a-transaction) should be
+accepted or rejected, it asks a small, random subset of validators whether they think the
+transaction should be accepted or rejected. If the queried validator thinks the transaction is
+invalid, has already rejected the transaction, or prefers a conflicting transaction, it replies that
+it thinks the transaction should be rejected. Otherwise, it replies that it thinks the transaction
```

Comment:
Could you do a pass to remove references to nodes responding that they think a transaction should be rejected?